src/Installer/AbstractModuleInstaller.php

```diff
@@ -230,16 +230,61 @@ protected function addSymlinks(PackageInterface $package, $targetRoot, array $pa
             }
         }
 
+        // Preserve the current working directory
+        $workingDir = getcwd();
+
         // Only actually create the links if the checks are successful to prevent orphans.
         foreach ($actions as $source => $target) {
             $this->logSymlink($source, $target);
 
             $this->filesystem->ensureDirectoryExists(dirname($target));
 
+            // Change directory because the source will be now relative
+            chdir(dirname($target));
+
+            $source = $this->convertPathToRelative($source, $target);
+
             if (!symlink($source, $target)) {
                 throw new \RuntimeException('Failed to create symlink ' . $target);
             }
         }
+
+        // Restore the previous working directory
+        chdir($workingDir);
+    }
+
+    /**
+     * Cover the path to be relative.
+     *
+     * @param string $source The source path.
+     *
+     * @param string $target The target path.
+     *
+     * @return string
+     */
+    protected function convertPathToRelative($source, $target)
```

Comment:
Maybe we should copy this from Symfony `Filesystem`? Pretty sure that code is mature and stable ;-)
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Filesystem/Filesystem.php#L435

Also, did you check if the Composer `Filesystem` does not provide such a method?